internal/clients/ionoscloud_test.go

```diff
@@ -15,23 +13,23 @@ import (
 
 	"github.com/ionos-cloud/crossplane-provider-ionoscloud/internal/clients/compute"
 	"github.com/ionos-cloud/crossplane-provider-ionoscloud/internal/clients/k8s"
-	"github.com/ionos-cloud/crossplane-provider-ionoscloud/internal/version"
 )
 
-const (
-	hostnameFromSecret = "https://host"
-	hostnameFromEnv    = "http://host-from-env"
-)
-
-func setComputeDefaults(cfg *ionos.Configuration) {
-	cfg.HTTPClient = http.DefaultClient
-	cfg.UserAgent = fmt.Sprintf("%v/%v_ionos-cloud-sdk-go/v%v", UserAgent, version.Version, ionos.Version)
-}
-
-func setDbaaSDefaults(cfg *ionosdbaas.Configuration) {
-	cfg.HTTPClient = http.DefaultClient
-	cfg.UserAgent = fmt.Sprintf("%v/%v_ionos-cloud-sdk-go-dbaas-postgres/v%v", UserAgent, version.Version, ionosdbaas.Version)
-}
+// TODO: Fix this tests in https://github.com/ionos-cloud/crossplane-provider-ionoscloud/issues/204
```

Comment:
why are those tests failing?

Reply:
Logger object in `ionoscloud.Configuration` is not comparable through DeepEqual